private/sass/addons/_events.scss

```diff
@@ -46,6 +46,9 @@ $events-visual-size: 240px;
         position: absolute;
         left: $line-height-computed / 1.5;
         top: $line-height-computed / 1.5;
+        bottom: $line-height-computed / 1.5;
+        overflow: hidden;
```

Comment:
doesn't this break the calendar:

![image](https://cloud.githubusercontent.com/assets/270307/14521653/a10ad178-022a-11e6-867d-272ad7dddc8e.png)

![image](https://cloud.githubusercontent.com/assets/270307/14521657/a9e3734a-022a-11e6-8a94-f82478ab8c2d.png)